### PR TITLE
draft of clarity and concision pass on sections 2.1 through 2.5

### PR DESCRIPTION
**Pull request by pes10k**
This is part of https://github.com/w3ctag/security-questionnaire/issues/107

I'm doing it in chunks to prevent horrible merge conflicts and to actually make progress

***
<a href="https://pr-preview.s3.amazonaws.com/w3ctag/security-questionnaire/pull/136.html" title="Last updated on Dec 17, 2021, 9:39 PM UTC (eca87b2)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3ctag/security-questionnaire/136/e1e9c93...eca87b2.html" title="Last updated on Dec 17, 2021, 9:39 PM UTC (eca87b2)">Diff</a>

### CODE REVIEW

File: index.bs [hunk 1]
```diff
@@ -336,52 +338,34 @@ to track a user
...
 reducing the uniqueness of values,
 rotating values,
 or otherwise making features no more identifying than is needed.
 <!-- https://github.com/w3ctag/design-principles/issues/215 -->
-Also, keep in mind that
-user agents make use of several different caching mechanisms.
-Which, if any, caches will store this new state?
-Are additional mitigations necessary?
 
-<div class=example>
```
Comment (pes10k) [inline index.bs:369]: I got rid of this example bc i think it doesn't seem to be demonstrating how persistent storage can be abused (it seems to be demonstrating how service workers can be abused, whether or not they persist across sessions

File: index.bs [hunk 2]
```diff
@@ -100,14 +100,14 @@ may be useful when doing so.
...
+  What information does this feature expose,
+  and for what purposes?
 </h3>
 
-User Agents should only expose information to the web
+User agents should only expose information to the Web
 when doing so is necessary to serve a clear user need.
-Does your feature expose information to origins?
-If so, how does exposing this information serve user needs?
+Does your feature expose information to wbe sites?
```
Comment (hober) [inline index.bs:109]: *web

File: index.bs [hunk 3]
```diff
@@ -160,17 +161,17 @@ mitigated the risk by reducing a policy's granularity after a redirect.
...
+  either?
 </h3>
 
 Personal information is any data about a user
 (for example, their home address),
 or information that could be used to identify a user,
 such as an alias, email address, or identification number.
 
-Note: Personal information is
+Personal information is
```
Comment (hober) [inline index.bs:174]: I think this should stay in a Note block.

**Review by hober — APPROVED** [review]
Found a couple of nits. Otherwise this looks great! Merge away once you've fixed the nits!

File: index.bs (hunk 2 above)
Comment (pes10k) [reply index.bs:109]: Look, if you're going to expect me to spell everything correctly, we're going to be at this for a long time :P

**Pull request merged**